package.json

```diff
@@ -15,9 +15,10 @@
     "start:prod": "node dist/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
     "test": "cross-env NODE_ENV=dev jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:watch": "cross-env NODE_ENV=dev jest --watch",
+    "test:cov": "cross-env NODE_ENV=dev jest --coverage",
+    "test:debug": "cross-env NODE_ENV=dev node --inspect-brk -r tsconfig-paths/register -r ts-node/register ./node_modules/jest/bin/jest.js --runInBand",
+    "test:debug:mac": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
```

Comment:
question: is cross-env not working on mac?

Reply:
cross-env should work for mac as well, the important difference it the changed path to the jest.js file